src/components/Carousel.js

```diff
@@ -398,6 +398,20 @@ export default class Carousel extends Component {
     this.resetInterval();
   };
 
```

Comment:
Maybe here would be convenient to add a description of this method as we have in the rest of all the others methods in this file. I think that here is even more useful because of the custom nature this method has. It would be harder to know in the future why we added a "simulatedEvent" without an introductory comment of this feature.